src/scripts/rechunk.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+
+import dask.array as da
+import argparse
+import time
+
+
+def timeit(method):
+    def timed(*args, **kw):
+        ts = time.time()
+        result = method(*args, **kw)
+        te = time.time()
+        if 'log_time' in kw:
+            name = kw.get('log_name', method.__name__.upper())
+            kw['log_time'][name] = int((te - ts) * 1000)
+        else:
+            print('%r  %2.2f ms' %
+                  (method.__name__, (te - ts) * 1000))
+        return result
+    return timed
+
+
+@timeit
+def load(source_array):
+    return da.from_zarr(source_array)
+
+
+@timeit
+def resize(arr, chunks):
+    return arr.rechunk(chunks)
+
+
+@timeit
+def convert(resized, target_array):
+    da.to_zarr(resized, target_array)
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--dimensions", type=int, default="5", metavar="DIMS",
+                        help="number of chunks to parse (%(default)s)")
+    parser.add_argument("--distributed", action="store_true",
+                        help="enable distributed dashboard (%(default)s)")
+    parser.add_argument("source_array",
+                        help="array to copy from. must exist.")
+    parser.add_argument("target_array",
+                        help="array to write to. may not exist.")
+    parser.add_argument("chunks", default="1024,1024,1,1,1",
```

Comment:
I expected to find the non-1's at the start, not the end.

Reply:
Ta.